nodes/materials/tree.py

```diff
@@ -163,7 +163,7 @@ def poll(cls, context):
         NodeItem("LuxCoreNodeTexIrregularData", label="Irregular Data"),
     ]),
 
-LuxCoreNodeCategoryMaterial("LUXCORE_MATERIAL_SHAPE", "Shape Modifiers", items=[
+    LuxCoreNodeCategoryMaterial("LUXCORE_MATERIAL_SHAPE", "Shape Modifiers", items=[

```

Comment:
While investigating the issue I noticed the indentation on this line was incorrect. This is not critical to solving the issue, but I thought it would be good to fix it anyway.